Sources/ProjectDescription/Dependencies/CarthageDependencies.swift

```diff
@@ -9,33 +9,45 @@ public struct CarthageDependencies: Codable, Equatable {
     /// Indicates whether Carthage produces XCFrameworks or regular frameworks. Refers to `--use-xcframeworks` Carthage flag.
     /// Note: It requires Carthage in version at least 0.37.0.
     public let useXCFrameworks: Bool
+    /// Indicates whether Carthage rebuilds the dependency from source instead of using downloaded binaries when possible. Refers to `--no-use-binaries` Carthage flag.
+    public let noUseBinaries: Bool
 
     /// Initializes a new `CarthageDependencies` instance.
     /// - Parameters:
     ///   - dependencies: List of depedencies that can be installed using Carthage.
     ///   - platforms: List of platforms for which you want to install depedencies. Refers to `--platform` Carthage flag.
     ///   - useXCFrameworks: Indicates whether Carthage produces XCFrameworks or regular frameworks. Refers to `--use-xcframeworks` Carthage flag. Note: It requires Carthage in version at least 0.37.0.
+    ///   - noUseBinaries: Indicates whether Carthage rebuilds the dependency from source instead of using downloaded binaries when possible. Refers to `--no-use-binaries` Carthage flag.
     init(
         dependencies: [Dependency],
         platforms: Set<Platform> = Set(Platform.allCases),
-        useXCFrameworks: Bool = false
+        useXCFrameworks: Bool = false,
+        noUseBinaries: Bool = false
     ) {
         self.dependencies = dependencies
         self.platforms = platforms
         self.useXCFrameworks = useXCFrameworks
+        self.noUseBinaries = noUseBinaries
     }
 
     /// Creates `CarthageDependencies` instance.
     /// - Parameters:
     ///   - dependencies: List of depedencies that can be installed using Carthage.
     ///   - platforms: List of platforms for which you want to install depedencies. Refers to `--platform` Carthage flag.
     ///   - useXCFrameworks: Indicates whether Carthage produces XCFrameworks or regular frameworks. Refers to `--use-xcframeworks` Carthage flag. Note: It requires Carthage in version at least 0.37.0.
+    ///   - noUseBinaries: Indicates whether Carthage rebuilds the dependency from source instead of using downloaded binaries when possible. Refers to `--no-use-binaries` Carthage flag.
     public static func carthage(
         _ dependencies: [Dependency],
         platforms: Set<Platform> = Set(Platform.allCases),
-        useXCFrameworks: Bool = false
+        useXCFrameworks: Bool = false,
+        noUseBinaries: Bool = false
```

Comment:
Yeah I could go both ways on adding an `unsafe` option or whatever here, but it's not like carthage doesn't have an already compiled set of flags in can take in.

Reply:
If we deem Carthage stable enough, which it probably is, we can just go ahead and copy all the flags instead of adding `.unsafeFlags()`, I think that could be a good option to choose